test/gtest/ucs/test_sys.cc

```diff
@@ -199,15 +199,25 @@ UCS_TEST_F(test_sys, memunits_to_str) {
 }
 
 UCS_TEST_SKIP_COND_F(test_sys, memcpy, RUNNING_ON_VALGRIND || !ucs::perf_retry_count) {
+#if UCS_ARCH_MEMCPY_RELAXED_SUPPORTED
```

Comment:
can we use runtime check if builtin_memcpy_min<builtin_memcpy_max in global opts?
instead of defining this macro

Reply:
the problem is in compilation: on non-x86 arch there is no such values in global_opt structure

Reply:
does it make sense to have gtest/ucs/arch/test_x86_64.cc?

Reply:
will do